### ISSUE TITLE
Document reusing storybook stories in tests

### ISSUE THREAD

**Issue by LarsDenBakker** [modernweb-dev/web#532]
We should document how to reuse storybook stories inside tests. For web components, but also using other test frameworks.

**Comment by yhjhoo** [comment]
I am looking for the way to integrate storybook into my testing? have you figured it out?

**Comment by IanVS** [comment]
@yhjhoo have you seen https://storybook.js.org/blog/interaction-testing-with-storybook/ and https://storybook.js.org/blog/stories-are-tests/?  There is a test runner if you write your tests into your stories, or you can use https://github.com/storybookjs/testing-react or https://github.com/storybookjs/testing-vue if you're using one of those frameworks.

**Comment by yhjhoo** [comment]
Yes, I checked these. However it doesn't have an example for web component example.
